addons/terrabrush/KeybindManager.cs

```diff
@@ -0,0 +1,161 @@
+    private readonly InputEventKey BrushSizeKey = new InputEventKey() { Keycode = Key.G };
+    private readonly InputEventKey BrushStrengthKey = new InputEventKey() { Keycode = Key.H };
+    private readonly InputEventKey EscapeKey = new InputEventKey() { Keycode = Key.Escape };
+    #endregion
+    
+    #region Public String Names
+
+    public static class StringNames
+    {
+	    public static readonly StringName ToolPie = new StringName("terrabrush_tool_pie");
+	    public static readonly StringName BrushPie = new StringName("terrabrush_brush_pie");
+	    public static readonly StringName ToolContentPie = new StringName("terrabrush_tool_content_pie");
+	    public static readonly StringName BrushSizeSelector = new StringName("terrabrush_brush_size_selector");
+	    public static readonly StringName BrushStrengthSelector = new StringName("terrabrush_brush_strength_selector");
+	    public static readonly StringName EscapeSelector = new StringName("terrabrush_brush_escape_selector");
+    }
+    #endregion
+    
+    // Quick access to Action Manes and Default Keys to register with the Godot Editor
+    public readonly List<StringName> ActionNames;
+    public readonly System.Collections.Generic.Dictionary<StringName, InputEventKey> DefaultKeys;
+    
+    public KeybindManager()
+    {
+	    ActionNames = new List<StringName>()
+	    {
+		    StringNames.ToolPie, StringNames.BrushPie, StringNames.ToolContentPie,
+		    StringNames.BrushSizeSelector, StringNames.BrushStrengthSelector, StringNames.EscapeSelector
+	    };
+	    DefaultKeys = new System.Collections.Generic.Dictionary<StringName, InputEventKey>()
+	    {
+		    { StringNames.ToolPie, ToolPieKey },
+		    { StringNames.BrushPie, BrushPieKey },
+		    { StringNames.ToolContentPie, ToolContentKey},
+		    { StringNames.BrushSizeSelector, BrushSizeKey },
+		    { StringNames.BrushStrengthSelector, BrushStrengthKey },
+		    { StringNames.EscapeSelector, EscapeKey }
+	    };
+    }
+
+    public Array<InputEvent> GetBinding(StringName actionName) => InputMap.Singleton.ActionGetEvents(actionName);
+
+    /// <summary>
+    /// Register's the Action Names with the Godot Editor InputMap.
+    /// </summary>
+    public void RegisterInputMap()
+    {
+	    var im = InputMap.Singleton;
+	    foreach (var action in ActionNames)
+	    {
+		    if (im.HasAction(action)) continue;	// We have the Action
+			im.AddAction(action); // Register the Action, as we don't have the Action.
+	    }
+    }
+    
+    
+    /// <summary>
+    /// Load Keybinds from Godot Editor's EditorSettings File, for association with the InputMap.Actions.
+    /// </summary>
+	public void LoadEditorSettings()
+	{
+		var settings = EditorInterface.Singleton.GetEditorSettings();
+		var im = InputMap.Singleton;
+		var shortcuts = settings.GetSetting("shortcuts").AsGodotArray<Godot.Collections.Dictionary>();
+		foreach (var action in ActionNames)
+		{
+			var binding = $"{SettingsGroup}/{action.ToString()}";
```

Comment:
```suggestion
			var binding = $"{SettingsGroup}/{action}";
```

It's already implicitly doing a ToString()